libenv/sysinfo.c

```diff
@@ -150,6 +150,8 @@ static time_t GetBootTimeFromUptimeCommand(time_t now);
 
 /*****************************************************/
 
+static char *FindNextInteger(char *str, char **num);
+
```

Comment:
This change means you have to update a line number in cppcheck_suppressions.txt:


```
// cppcheck is not clever enough to see that if (i >= PLATFORM_CONTEXT_MAX) then 'found' is false
arrayIndexOutOfBounds:libenv/sysinfo.c:585
```